perfkitbenchmarker/providers/gcp/gcp_tpu.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module containing class for GCP's cloud TPU.
+
+cloud TPU can be created and deleted.
+"""
+
+import json
+import logging
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import resource
+from perfkitbenchmarker.providers.gcp import util
+
+DEFAULT_CLOUD_SPANNER_CONFIG = 'regional-us-central1'
+TPU_TIMEOUT = 1200
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('tpu_version', 'nightly',
+                    'The version for the Cloud TPU.')
+
+
+class GcpCloudTpu(resource.BaseResource):
```

Comment:
I'm considering creating an 'Accelerator' base resource so that we can specify both GPUs and TPUs in the same manner, but I'm not completely sold. If not, then we should add TPUs to GceVmSpec, so that we specify TPUs in the benchmark config, like we do with GPUs.

Reply:
I don't know. I think creating TPU is isolated with creating a GCE. 
Creating GCE using command "gcloud compute instances create"
Creating TPU using command "gcloud alpha compute tpus create"
If the TPU is created successfully, then any GCE VMs can use this TPU, just using the IP of this TPU.
That's why I created this TPU like this.

Reply:
Ok, let's leave it out of GceVmSpec. I still think we should add it to the benchmark_config_spec to be consistent with PKB. I can help with this.

Reply:
Thanks. I will add it to benchmark_config_spec.